runbotics/runbotics-orchestrator-ui/src/src-landing/scss/variables/_spacing.scss

```diff
@@ -1,19 +1,19 @@
-$margin-0: 0rem;
-$margin-1: 0.5rem;
-$margin-2: 1rem;
-$margin-3: 1.5rem;
-$margin-4: 2rem;
-$margin-5: 2.5rem;
-$margin-6: 3rem;
-$margin-7: 3.5rem;
-$margin-8: 4rem;
+$margin-0: 0px;

```

Comment:
let's discuss this px vs rem in padding

Reply:
https://medium.com/@sascha.wolff/dont-use-rem-em-for-paddings-margins-and-more-94e19026b000